activesupport/lib/active_support/benchmark.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module ActiveSupport
+  # = \Benchmark
+  module Benchmark
+    # Benchmark realtime in seconds.
+    #
+    #   ActiveSupport::Benchmark.realtime { User.expensive_query.count }
+    #   # => 5.8273
+    def self.realtime(&block)
+      time_start = Process.clock_gettime(Process::CLOCK_MONOTONIC)
+      yield
+      Process.clock_gettime(Process::CLOCK_MONOTONIC) - time_start
+    end
+
+    # Benchmark realtime in milliseconds.
+    #
+    #   ActiveSupport::Benchmark.realtime { User.all }
+    #   # => 8.0e-05
+    #
+    #   ActiveSupport::Benchmark.ms { User.all }
+    #   # => 0.074
+    def self.ms(&block)
+      1000 * realtime(&block)
```

Comment:
It would be better if a method would be implemented in https://github.com/ruby/benchmark. Do you think we should try a PR?